packages/react/src/SharedComponents/ColorPicker/components/ColorPicker.tsx

```diff
@@ -5,77 +5,95 @@ import { TiTick } from "react-icons/ti";
 export const ColorPicker = (props: {
   onClick?: () => void;
   iconSize?: number;
-  textColor: string;
-  setTextColor: (color: string) => void;
-  backgroundColor: string;
-  setBackgroundColor: (color: string) => void;
+  text?: {
```

Comment:
I made it so that the color picker could display either text or background colors, or both. Before it could only display both. So I made this change since otherwise the typing would be wonky (`textColor`, `setTextColor`, `backgroundColor`, and `setBackgroundColor` would all have to be optional, less clear what you're supposed to pass).